src/pages/Rockets.js

```diff
@@ -1,11 +1,28 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Rocket from '../components/Rocket';
+import { fetchRockets } from '../redux/rockets/rocket';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets);
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
+  return (
+    <main className="d-flex flex-column align-center col-md-12">
+      <div className="container rockets d-flex flex-column">
```

Comment:
Rockets is the default view, so you must fetch rockets data when the application starts.
You can make some adjustments in the routes and have something like this for the rocket's `route`
`<NavLink  to="/" />`